packages/react-native-bridge/ios/RNReactNativeGutenbergBridge.swift

```diff
@@ -374,6 +374,7 @@ extension RNReactNativeGutenbergBridge {
         case updateCapabilities
         case showNotice
         case mediaSave
+		case showEditorHelp
```

Comment:
It looks like the indentation may be off on this line in comparison to other lines in this file. Not sure if GitHub is presenting it odd or if it is truly off.

Reply:
Not sure why github is displaying it this way, but I opened the file in VS Code and it's definitely not formatted that way: 

![Screen Shot 2021-07-02 at 5 01 42 PM](https://user-images.githubusercontent.com/5810477/124328197-30112600-db57-11eb-91ae-f2544a005a2f.png)

Reply:
@AmandaRiu it appears to be the difference between tabs vs spaces. Xcode appears to default to spaces. Modifying this file with VSCode likely led to the discrepancy. I'm not sure the team's stance on tabs vs space (or if it matters at all), but it does appear the majority of the Swift files use spaces.

Reply:
@dcalhoun you were correct about the spaces vs. tabs. I updated in 5189548855c49f940ceb2bc406dd0b2203c144c6 and that seems to have fixed it.